web/src/search/saved-queries/SavedQueryForm.tsx

```diff
@@ -77,7 +77,11 @@ export class SavedQueryForm extends React.Component<Props, State> {
                     filter(isSettingsValid)
                 )
                 .subscribe(settingsCascade => {
-                    const subject = settingsCascade.subjects.find(s => !!s.subject.id)
+                    const subject = settingsCascade.subjects.find(
+                        s =>
+                          !!s.subject.id &&
+                          (s.subject.__typename === "Org" || s.subject.__typename === "User")
```

Comment:
```suggestion
                          s.subject.viewerCanAdminister
```

This should be changed to check whether the current user is able to edit the subject by checking the `viewerCanAdminister` property of the subject.  Users that are site admins are actually able to create global saved queries by not selecting any save location, so we don't want to filter for only Org and User settings (This is bad UI that is changing soon).

Reply:
I get it! Thanks, @attfarhan.